app/src/main/java/com/realityexpander/tasky/core/presentation/common/util/UiText.kt

```diff
@@ -245,6 +245,22 @@ open class UiText : Parcelable {
         }
     }
 
+    // Useful for plain kotlin classes that don't have access to @Composable functions.
+    // Returns null if Str is `None` or `Str` is null value. Ignores the Resource value
+    // (good for testing)
+    fun asStrOrNull(): String? {
+        return when(this) {
+            is None -> null
```

Comment:
Why not make the `UiText` nullable and get rid of `None`?

Reply:
I will remove it